trace_replay/block_cache_tracer.cc

```diff
@@ -266,4 +280,16 @@ Status BlockCacheTracer::WriteBlockAccess(const BlockCacheTraceRecord& record,
                                           referenced_key);
 }
 
+uint64_t BlockCacheTracer::NextGetId() {
+  if (!writer_.load(std::memory_order_relaxed)) {
+    return 0;
+  }
+  InstrumentedMutexLock lock_guard(&get_id_counter_mutex_);
+  if (get_id_counter_ == port::kMaxUint64) {
```

Comment:
Can you implement this with atomic variable?

Reply:
Sure. I removed the mutex and updated to use atomic variable.